app/src/main/java/app/grapheneos/camera/ui/activities/MainActivity.kt

```diff
@@ -203,6 +210,13 @@ open class MainActivity : AppCompatActivity(),
         startFocusTimer()
     }
 
+    private fun shouldUseFocusRegion(): Boolean {
+        //OnePlus 6/6T: https://github.com/GrapheneOS/Camera/issues/204
```

Comment:
Is it worth it to cache this value?

Reply:
You should be able to replace the function with a field. Can also use `when` for pattern matching since we're going to need to add the other OnePlus 6 and 6T model names in addition to these 2. I'm unsure where to get a full list. I could see if the Play Store has a list.

```
val shouldUseFocusRegion = when (Build.MODEL) {
    "ONEPLUS A6003", "ONEPLUS A6013" -> true
    else -> false
}
```

Reply:
Did that backwards, fixed.

Reply:
Is there a field we could use that's set to `OnePlus6` instead? Maybe Build.PRODUCT instead of Build.MODEL?

Reply:
That works:
https://github.com/GrapheneOS/AttestationSamples/blob/main/ONEPLUS%20A6003/filtered.prop#L43
https://github.com/GrapheneOS/AttestationSamples/blob/main/ONEPLUS%20A6013/filtered.prop#L46